Source/SwiftLintFramework/Rules/OpeningBraceRule.swift

```diff
@@ -25,16 +36,89 @@ public struct OpeningBraceRule: Rule {
             "func abc()\n\t{ }",
             "[].map(){ $0 }",
             "[].map( { } )"
+        ],
+        corrections: [
+            "struct Rule{}\n": "struct Rule {}\n",
+            "struct Rule\n{\n}\n": "struct Rule {\n}\n",
+            "struct Rule\n\n\t{\n}\n": "struct Rule {\n}\n",
+            "struct Parent {\n\tstruct Child\n\t{\n\t\tlet foo: Int\n\t}\n}\n":
+                "struct Parent {\n\tstruct Child {\n\t\tlet foo: Int\n\t}\n}\n",
+            "[].map(){ $0 }\n": "[].map() { $0 }\n",
+            "[].map( { } )\n": "[].map({ } )\n",
```

Comment:
Should we *not* correct this example? 
(It so, `swiftlint autocorrect` will not improve this warning.)